src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -173,17 +190,57 @@ namespace ompl
                 return pruneStatesThreshold_;
             }
 
+            /** \brief Use a k-nearest search for rewiring instead of a r-disc search. */
+            void setKNearest(bool useKNearest)
+            {
+                useKNearest_ = useKNearest;
+            }
+
+            /** \brief Get the state of using a k-nearest search for rewiring. */
+            bool getKNearest() const
+            {
+                return useKNearest_;
+            }
+
```

Comment:
Ok, just leave as-is. I thought there might have been an opportunity to unify some code.